debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlDdlParserTest.java

```diff
@@ -366,20 +367,21 @@ public void shouldParseSetCharacterSetStatement() {
         assertVariable("character_set_connection", null);
         assertVariable("character_set_database", null);
 
-        parser.parse("CREATE DATABASE db1 CHARACTER SET cs1;", tables);
+        parser.parse("CREATE DATABASE db1 CHARACTER SET LATIN1;", tables);
```

Comment:
Why is this changing?

Reply:
There is no cs1 character set. The ANTLR MySQL grammar is expecting the existing charsets (you can see the list of them in `charsetNameBase` rule in grammar).